src/soundsourcemp3.cpp

```diff
@@ -515,6 +478,10 @@ unsigned SoundSourceMp3::read(unsigned long samples_wanted, const SAMPLE * _dest
                 break;
             }
         }
+        if (frameBefore == Stream->this_frame) {
+            // No seek, try again
+            continue;
```

Comment:
@uklotzde: I am currently debugging, why this is required. It happens always after a seek. 
No result until now :-(
This is probably a bug in our init code, or something that triggers a mad bug.